content/blog/modern-text-features/index.Rmd

```diff
@@ -0,0 +1,381 @@
+---
+output: hugodown::hugo_document
+
+slug: modern-text-features
+title: Modern Text Features in R
+date: 2021-02-06
+author: Thomas Lin Pedersen
+description: >
+  ragg has taken a major leap forward in text rendering capabilities with the
+  latest releases of systemfonts, textshaping, and ragg itself. This post will
+  go into detail with what is now possible and how it compares to the build in 
+  devices.
+
+photo:
+  url: https://unsplash.com/photos/bMybTSV7RFY
+  author: Natalia Y
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [deep-dive] 
+tags: [graphic-device, ragg, systemfonts]
+---
+
+```{r setup, include=FALSE}
+library(ragg)
+
+knitr::opts_chunk$set(
+  dev = "ragg_png"
+)
+
+set.seed(1)
+```
+
+
+I'm extremely pleased to present the culmination of several years of work on font rendering, spanning the [systemfonts](https://github.com/r-lib/systemfonts), [textshaping](https://github.com/r-lib/textshaping), and [ragg](https://ragg.r-lib.org) packages. These releases complete our efforts to create a high-quality, performant raster graphics device that works comparably on every operating system.
+
+The result of the latest work is that font rendering in the ragg graphics devices now just works, regardless of what you throw at it.
+
+This includes:
```

Comment:
```suggestion
I'm extremely pleased to present the culmination of several years of work spanning the [systemfonts](https://github.com/r-lib/systemfonts), [textshaping](https://github.com/r-lib/textshaping), and [ragg](https://ragg.r-lib.org) packages. These releases complete our efforts to create a high-quality, performant raster graphics device that works the same way on every operating system.

This blog post presents our improvements to ragg's font rendering so that it now "just works" regardless of what you throw at it.  This includes:
```